system/web/context/ExceptionBean.cfc

```diff
@@ -41,6 +41,20 @@ component accessors="true"{
 		}
 		variables.extraMessage 	= arguments.extraMessage;
 		variables.extraInfo 	= arguments.extraInfo;
+		
+		try {
+			if(structIsEmpty(variables.exceptionStruct) && isObject(arguments.errorStruct)){
```

Comment:
@sanaullah if we get here, we know it is this special case, why not just duplicate it instead:
`variables.exceptionStruct = duplicate( arguments.errorStruct )`